### PR TITLE
add new binary pluralctl

### PR DESCRIPTION
**Pull request by zreigz**
## Summary
A new distribution of the CLI for fleet-management.

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: cmd/pluralctl/main.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	app.Name = ApplicationName
+	app.Usage = "Tooling to manage and operate a fleet of clusters"
+	app.EnableBashCompletion = true
+	app.Flags = globalFlags()
+	commands := []cli.Command{
+		cryptocmd.Command(plural.Plural),
+		cd.Command(plural.Plural, plural.HelmConfiguration),
+		up.Command(plural.Plural),
+		pr.Command(),
+		stack.Command(plural.Plural),
```
Comment (michaeljguarino) [inline cmd/pluralctl/main.go:99]: Are these stacks from console or plural?  We'd only care about the console stack commands here, and I do think they deserve a separate subcommand.

File: cmd/pluralctl/main.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+		cmdinit.Command(plural.Plural),
+		mgmt.Command(plural.Plural),
```
Comment (michaeljguarino) [inline cmd/pluralctl/main.go:101]: We should probably also keep:

* `profile` commands
* `clone` command (to help clone + decrypt repos)
* `version` command (always useful to have)

File: cmd/pluralctl/main.go (hunk 1 above)
Comment (zreigz) [reply cmd/pluralctl/main.go:99]: those are plural stacks. We have console stacks in `cd` subcommand

Comment (michaeljguarino) [reply cmd/pluralctl/main.go:99]: Yeah in that case transfer out the ones from `cd` into the top-level here if possible.

Comment (michaeljguarino) [reply cmd/pluralctl/main.go:99]: the idea is the cli should sort of mimic the sidebar tab structure in the console ui.

Comment (zreigz) [reply cmd/pluralctl/main.go:99]: I will try to do this

**Comment by zreigz** [comment]
@michaeljguarino 
I've moved CD commands to the top level.
I had to move the plural login to the management group commands (duplicated with the CD login)

File: cmd/pluralctl/main.go (hunk 2 above)
Comment (michaeljguarino) [reply cmd/pluralctl/main.go:72]: We do still want a top-level `login` command, not sure if it's here still from the pr (maybe cmdinit brings it in?)

**Comment by michaeljguarino** [comment]
> @michaeljguarino I've moved CD commands to the top level. I had to move the plural login to the management group commands (duplicated with the CD login)

We should still keep the `cd` commands under the `cd` subcommand I think, that'll prevent the need to move the login command as well.

File: cmd/pluralctl/main.go (hunk 2 above)
Comment (zreigz) [reply cmd/pluralctl/main.go:72]: yes, the `plural init` can handle login.

**Comment by zreigz** [comment]
CD commands are in `cd` group again. The `plural init` can handle the login part, PTAL

**Pull request merged**